src/frontend/screens/MapScreen/index.tsx

```diff
@@ -74,6 +59,10 @@ export const MapScreen = () => {
     setFollowing(true);
   }
 
+  if (savedLocation.isLoading) {
+    return <Loading />;
+  }
```

Comment:
does the whole screen need to be blocked by this? what happens if this is omitted?

Reply:
its seems to be fine if we omit it. Addressed here: https://github.com/digidem/CoMapeo-mobile/pull/133/commits/e7a70b87055e9662767ad04ad3e00327e11d1575